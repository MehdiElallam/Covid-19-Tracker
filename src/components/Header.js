import React from 'react'
import styled from 'styled-components';


export default function Header({title}) {
    return (
        <Wrapper>
            <h1>{title}</h1>
            <p>The official UK government website for data and insights on coronavirus (COVID-19).</p>
        </Wrapper>
    )
}

const Wrapper = styled.div`
    background: #000;
    width: 100%;
    margin-bottom : 20px;
    padding: 15px 25px;
    color: #FFF;
`;