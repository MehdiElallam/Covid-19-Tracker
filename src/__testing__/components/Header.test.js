import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

describe('Header testing', () => {
  
  it('Header should render correctly', () => {
    const component = shallow(<Header />);
  
    expect(component).toMatchSnapshot();
  });
});