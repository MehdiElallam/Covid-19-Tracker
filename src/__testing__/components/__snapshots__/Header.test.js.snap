// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`Header testing Header should render correctly 1`] = `ShallowWrapper {}`;
